ibm/service/sdsaas/resource_ibm_sds_host.go

```diff
@@ -39,71 +39,61 @@ func ResourceIBMSdsHost() *schema.Resource {
 				Type:         schema.TypeString,
 				Optional:     true,
 				ValidateFunc: validate.InvokeValidator("ibm_sds_host", "name"),
-				Description:  "The name for this host. The name must not be used by another host.  If unspecified, the name will be a hyphenated list of randomly-selected words.",
+				Description:  "Unique name of the host.",
 			},
 			"nqn": &schema.Schema{
 				Type:         schema.TypeString,
 				Required:     true,
 				ValidateFunc: validate.InvokeValidator("ibm_sds_host", "nqn"),
-				Description:  "The NQN of the host configured in customer's environment.",
+				Description:  "The NQN (NVMe Qualified Name) as configured on the initiator (compute/host) accessing the storage.",
 			},
-			"volumes": &schema.Schema{
+			"volume_mappings": &schema.Schema{
```

Comment:
renaming the arguments will be breaking change. Follow these guidelines for the process
https://developer.hashicorp.com/terraform/plugin/framework/deprecations